Mathlib/SetTheory/ZFC/Basic.lean

```diff
@@ -184,6 +184,15 @@ theorem Subset.congr_right : ∀ {x y z : PSet}, Equiv x y → (z ⊆ x ↔ z 
       let ⟨a, ab⟩ := βα b
       ⟨a, cb.trans (Equiv.symm ab)⟩⟩
 
+instance : Preorder PSet where
+  le := (· ⊆ ·)
+  le_refl := refl_of (· ⊆ ·)
+  le_trans _ _ _ := trans_of (· ⊆ ·)
```

Comment:
Would you define the HasSSubset instance too? I guess the auto-generated `lt` would be exactly how one would define ⊂, but I think we still want to use ⊆ and ⊂ instead of ≤ and < as simp normal forms as in the case of Set/Finset?